types.go

```diff
@@ -234,6 +237,53 @@ type VirtualMachine struct {
 	HA             HA         `json:",omitempty"`
 }
 
+type Nic struct {
```

Comment:
if it's an acronym try and use all caps, NICs and NIC

Reply:
Alright, I was wondering while browsing through the code. Never did it like that haha..

Reply:
just a go thing that comes in from the official style guide. https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Ahh I see, will look at it, Thanks!